modules/hcp-custom-cert.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-custom-cert_{context}"]
+= Configuring a custom API server certificate in a hosted cluster
+
+To configure a custom certificate for the API server, specify the certificate details in the `spec.configuration.apiServer` section of your `HostedCluster` configuration. 
+
+You can configure a custom certificate during either day-1 or day-2 operations. However, because the service publishing strategy is immutable after you set it during hosted cluster creation, you must know what the hostname is for the Kubernetes API server that you plan to configure.
+
+.Prerequisites
+
+* You created a Kubernetes secret that contains your custom certificate in the management cluster. The secret contains the following keys:
+
+ ** `tls.crt`: The certificate
+ ** `tls.key`: They private key
+
+* If your `HostedCluster` configuration includes a service publishing strategy that uses a load balancer, ensure that the Subject Alternative Names (SANs) of the certificate do not conflict with the internal API endpoint (`api-int`). The internal API endpoint is automatically created and managed by your platform. If you use the same hostname in both the custom certificate and the internal API endpoint, routing conflictcs can occur. The only exception to this rule is when you use {aws-short} as the provider with either Private or PublicAndPrivate configurations. In those cases, the SAN conflict is managed by the platform.
+
+* The certificate must be valid for the external API endpoint.
+
+* The validity period of the certificate aligns with your cluster's expected life cycle.
+
+.Procedure
+
+. Create a secret with your custom certificate by entering the following command:
++
+[source,terminal]
+----
+$ oc create secret tls sample-hosted-kas-custom-cert \
+  --cert=path/to/cert.crt \
+  --key=path/to/key.key \
+  -n <hosted_cluster_namespace>
+----
+
+. Update your `HostedCluster` configuration with the custom certificate details, as shown in the following example:
++
+[source,yaml]
+----
+spec:
+  configuration:
+    apiServer:
+      servingCerts:
+        namedCertificates:
+        - names: <1>
+          - api-custom-cert-sample-hosted.sample-hosted.example.com
+          servingCertificate: <2>
+            name: sample-hosted-kas-custom-cert
+----
+<1> The list of DNS names that the certificate is valid for.
+<2> The name of the secret that contains the custom certificate.
+
+. Apply the changes to your `HostedCluster` configuration by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f <hosted_cluster_config>.yaml
+----
+
+.Verification
+
+* Check the API server pods to ensure that the new certificate is mounted.
+
```

Comment:
Can we add specific instructions here for how to check the pods to ensure that the certificate is mounted?